### PR TITLE
WW-5301 Fix custom VelocityManager bean selection

### PR DESCRIPTION
**Pull request by kusalk**
WW-5301
--

### CODE REVIEW

File: core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java [hunk 1]
```diff
@@ -109,6 +108,8 @@ public class Dispatcher {
 
     public static final String MULTIPART_FORM_DATA_REGEX = "^multipart/form-data(?:\\s*;\\s*boundary=[0-9a-zA-Z'()+_,\\-./:=?]{1,70})?(?:\\s*;\\s*charset=[a-zA-Z\\-0-9]{3,14})?";
 
+    private static final String CONFIG_SPLIT_REGEX = "\\s*,\\s*";
```
Comment (lukaszlenart) [inline core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java:111]: 👏

File: core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java [hunk 2]
```diff
@@ -554,6 +566,7 @@ public void init() {
             init_CustomConfigurationProviders(); // [5]
             init_FilterInitParameters(); // [6]
             init_AliasStandardObjects(); // [7]
+            init_DeferredXmlConfigurations();
```
Comment (lukaszlenart) [inline core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java:569]: Nice, do you plan document this functionality in the docs?

**Review by lukaszlenart — COMMENTED** [review]
Looks good, one question

File: core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java (hunk 2 above)
Comment (kusalk) [reply core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java:569]: I will create a struts-site PR adding a note about `struts-deferred.xml` to:
- https://struts.apache.org/plugins/plugins-architecture
- https://struts.apache.org/core-developers/struts-xml

**Review by lukaszlenart — APPROVED** [review]
LGTM 🎉

**Pull request merged**